pkg/helper/client.go

```diff
@@ -214,33 +214,37 @@ func checkApiServerConnectivity(timeout time.Duration) error {
 		config.Timeout = timeout
 		client, err := client.New(config, client.Options{})
 		if err != nil {
-			return false, errors.Wrap(err, "creating new custom client")
+			log.Error(err, "failed to creating new custom client")
+			return false, nil
 		}
 		err = client.Get(context.TODO(), types.NamespacedName{Name: metav1.NamespaceDefault}, &corev1.Namespace{})
 		if err != nil {
-			return false, errors.Wrap(err, "reaching to apiserver failed")
+			log.Error(err, "failed reaching the apiserver")
+			return false, nil
 		}
 		return true, nil
 	})
 }
 
 func defaultGw() (string, error) {
 	defaultGw := ""
-	return defaultGw, wait.PollImmediate(1*time.Second, defaultGwRetrieveTimeout, func() (bool, error) {
+	return defaultGw, wait.PollImmediate(1*time.Second, defaultGwRetrieveTimeout*time.Second, func() (bool, error) {
```

Comment:
maybe the timeout constant should be initialized with the unit as well

Reply:
Ahh right, let me do that thanks !